packages/launch/src/focused-launch/domain-details/test/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { MemoryRouter as Router } from 'react-router-dom';
+import { render, screen } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import DomainStep from '../';
+
+describe( 'DomainStep', () => {
+	test( 'Has title and sub-title', () => {
+		render(
+			<Router>
+				<DomainStep />
+			</Router>
+		);
+
+		expect( screen.queryByText( /Choose a domain/i ) ).toBeTruthy();
+		expect( screen.queryByText( /Free for the first year with any paid plan/i ) ).toBeTruthy();
+	} );
+
+	test( 'Has domain search input', () => {
+		render(
+			<Router>
+				<DomainStep />
+			</Router>
+		);
+
+		expect( screen.queryByPlaceholderText( /Search for a domain/i ) ).toBeTruthy();
+	} );
+
+	test( 'Has a back button', () => {
+		render(
+			<Router>
+				<DomainStep />
+			</Router>
+		);
+
+		expect( screen.queryByText( /Go back/i ) ).toBeTruthy();
+	} );
+} );
```

Comment:
It's really nice to see some tests ❤️ 
Of course, we can add more with time (eg: "can set domain", "can unset domain", "can fire analytics", etc) but having a starting point is more than welcome!

Reply:
Yeah, I noticed there was no tests in this package so this was more of an opportunity to implement the `jest.config.js` and the `@testing-library/react` package to enable us all to start writing them. Obviously they're rather simple tests but better than nothing right now I think 😅

Reply:
Please add more specs to #46943 , so that we can keep track of all the tests we want to write